Data/DolphinTextureHash.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+namespace DolphinDynamicInputTextureCreator.Data
+{
+    /// <summary>
+    /// Gives specific information about the Dolphin textures hash.
+    /// </summary>
+    public class DolphinTextureHash
+    {
+        public enum ImageFormat { I4, I8, IA4, IA8, RGB565, RGB5A3, RGBA32, Unknown, C4, C8, C14X2, CMPR = 14 }
```

Comment:
Can we specify these explicitly and remove the 'Unknown'?  I know you currently use it when it's invalid, maybe we can have this be a nullable instead?

The value isn't necessarily unknown on Dolphin's side (I can't say) but it'd be better if it matches near 1:1 with Dolphin's information..